### DISCUSSION TITLE
In VS code extension is installed but no icon appears in the bottom status bars area.

### DISCUSSION THREAD

**Discussion by RSUIAutomation** [community/community#6861] [Copilot Conversations]
As the instructions indicate, a GitHub copilot icon should appear, but nothing has.

**Comment by davecheney** [comment]
Are you behind a firewall, VPN, or other device that may be intercepting your HTTPS connections?

**Comment by RSUIAutomation** [comment]
No I think it's vscode blocking it but vscode docs dont say anything about what to do:

![Screen Shot 2021-10-28 at 10 06 48 AM](https://user-images.githubusercontent.com/79863650/139284272-ce70e0d1-9b14-4f2a-89d9-29b78da13a65.png)

**Reply by davecheney** [reply]
Which version of vscode are you using. Can you paste a screenshot of the about visual studio pop up?

**Comment by jacobcoro** [comment]
I'm having this same problem. I'm in China btw and use VPN sometimes

**Comment by jacobcoro** [comment]
It is not working with or without vpn

**Comment by jacobcoro** [comment]
<img width="497" alt="Screen Shot 2022-02-11 at 5 49 15 PM" src="https://user-images.githubusercontent.com/52448067/153570288-37f77dfe-f3db-4423-8e15-3c6fe749b17e.png">
<img width="752" alt="Screen Shot 2022-02-11 at 5 49 07 PM" src="https://user-images.githubusercontent.com/52448067/153570297-4e9065d8-1bb3-49a5-bebc-697b7e86a005.png">

**Comment by jacobcoro** [comment]
'configuration.icons' ids can only contain letter, digits and minuses and need to consist of at least two segments in the form `component-iconname`.
